jobs/config.json

```diff
@@ -221,9 +221,17 @@
   },
   "ci-ingress-gce-upgrade-e2e": {
     "args": [
-      "true"
+      "--check-leaked-resources",
+      "--cluster=",
+      "--extract=ci/latest",
```

Comment:
Last time I tried, we will need to specify `--extract` twice, meaning upgrade from one to the another.

Reply:
Although we only need the latest branch, there is an unit test ensuring that...

Reply:
@krzyzacy Are you aware of this? Any other special things we need to consider?

Reply:
Our test is only swapping out the ingress image so we don't need two extract's. Seems like it breaks the unit test though so I will add it.